pvdeg/temperature.py

```diff
@@ -41,19 +44,66 @@ def module(
 
     if poa is None:
         poa = pvdeg.spectral.poa_irradiance(weather_df, meta)
+    if "wind_height" not in meta.keys():
+        wind_speed_factor = 1
+    else:
+        if temp_model == "sapm":
+            wind_speed_factor = (10 / float(meta["wind_height"])) ** wind_factor
+        elif temp_model == "pvsyst":
+            wind_speed_factor = (2 / float(meta["wind_height"])) ** wind_factor
+        elif temp_model == "faiman":
+            wind_speed_factor = (2 / float(meta["wind_height"])) ** wind_factor
+        elif temp_model == "faiman_rad":
+            wind_speed_factor = (2 / float(meta["wind_height"])) ** wind_factor
+        elif temp_model == "fuentes":
+            wind_speed_factor = (5 / float(meta["wind_height"])) ** wind_factor
+        elif temp_model == "ross":
+            wind_speed_factor = (
+                10 / float(meta["wind_height"])
+            ) ** wind_factor  # I had to guess what this one was
+        elif temp_model == "notc_sam":
+            if meta["wind_height"] > 3:
+                wind_speed_factor = 2
+            else:
+                wind_speed_factor = (
+                    1  # The wind speed height is managed weirdly for this one.
+                )
+        elif temp_model == "prilliman":
```

Comment:
Note that the prilliman model is different from the others; rather that modeling a steady-state cell temperature as SAPM, faiman, etc do, it converts a modeled steady-state cell temperature into a transient cell temperature.  So it's a post-processing step on the other models.  I don't know the context of this code, I just thought it was unusual to see prilliman listed as an alternative to the other models, since conceptually it is not the same species of model as the others.

Reply:
Good catch. At this point I just put in placeholders for the different temperature models as listed in pvlib. So It sounds like we should just delete that as a future option. So is calling it directly from pvlib the appropriate thing to do? or does it make sense to write code in PVdeg to make its use easier? Would code in pvdeg be a stand alone method?

Reply:
I'm not familiar with PVdeg at all, I just happened to look at this PR :) So I can't comment on what is suitable for PVdeg to do.

One tidbit I can offer is that the Prilliman model only takes effect in simulations with time step < 20 minutes, which might make it wholly irrelevant if the focus here is on hourly data.

Reply:
We don't know what people will want to do at this time. But leaving it open for people to work on and implement later is reasonable. Long term someone will want to do an interpolation of the data. I'd have to look into this model to see if it is something I would want to use. None of these models really capture the transients properly anyway.

Reply:
We are not constrained to hourly data, but we can add a check so it's only an option for sub-hourly data of 20 or less intervals

Reply:
Presently, it is just a place holder. How about we just leave it in and whoever works on it to get this model working can determine what is best.